src/utils/getConfig.ts

```diff
@@ -2,6 +2,12 @@ import type { AvifOptions, JpegOptions, PngOptions, WebpOptions } from 'sharp'
 
 import type { AllowedFormat } from '../cli/utils/formatValidate'
 
+export type ParsedImageInfo = {
```

Comment:
Do you use this type elsewhere?
If not, how about removing `export`?

Reply:
We do use it within image.tsx.

Reply:
Oh, sorry.
I missed it.

Reply:
I'll keep the export in order to use the type in image.tsx